changes/10888-backend-analytics-for-premium

```diff
@@ -0,0 +1 @@
+* Removed the ability for Premium customers to disable usage statistics. The type of data collected has not changed; please see https://fleetdm.com/docs/using-fleet/usage-statistics for information on the exact contents of the usage statistics data.
```

Comment:
We should double check https://fleetdm.com/docs/using-fleet/usage-statistics is up-to-date (maybe we added some fields recently?)